src/adaPets.js

```diff
@@ -9,29 +9,54 @@ const BASE_URL = 'http://localhost:3000/pets/';
 
 // Option functions.
 const listPets = () => {
-  // Fill out as part of Wave 1.
+  axios.get(BASE_URL)
+  .then( (response) => {
+    setResult(response.data);
+  })
+  .catch( (error) => {
+    setError(`ENCOUNTERED PROBLEM: \" ${error} \"`);
+  });
 };
 
 const showDetails = (selectedPetId) => {
   if (!selectedPetId) {
     setError("You tried to show details for a pet without selecting it!");
-    
   }
-
-  // Fill out as part of Wave 2.
+  else {
+    axios.get(`http://localhost:3000/pets/${selectedPetId}`)
```

Comment:
You should have used `BASE_URL` here:

```suggestion
    axios.get(`${BASE_URL}${selectedPetId}`)
```